sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -246,7 +246,7 @@ def test_session_by_session_client_conn_str_receive_handler_with_inactive_sessio
             with sb_client.get_queue_session_receiver(servicebus_queue.name, 
                                               session_id=session_id, 
                                               mode=ReceiveSettleMode.ReceiveAndDelete, 
-                                              idle_timeout=5) as session:
+                                              max_wait_time=5) as session:
                 for message in session:
                     messages.append(message)
 
```

Comment:
sorry I can't comment directly on line 253,
but the assertion part of this case should be `assert session._running` instead of `assert not session._running`.

Reply:
heh just got that in the last pass, but good catch.